django/db/backends/base/base.py

```diff
@@ -47,6 +47,7 @@ class BaseDatabaseWrapper:
     introspection_class = None
     ops_class = None
     validation_class = BaseDatabaseValidation
+    is_pool = False
```

Comment:
From what I can tell, this is mostly used for skipping tests
Have you looked into whether those tests can have the connection settings updated in the test (similar to `tests.backends.postgresql.tests.Tests.test_connect_pool_set_to_true` for example) so that we have some test coverage in our current ci and we don't impact the base backend?

Reply:
I have removed `is_pool` from the base backend and made the test similar to `tests.backends.postgresql.tests.Tests.test_connect_pool_set_to_true`.